client_test.go

```diff
@@ -136,10 +147,6 @@ func TestMain(m *testing.M) {
 			fmt.Fprintln(os.Stderr, "failed to remove test root dir", err)
 			os.Exit(1)
 		}
-		// only print containerd logs if the test failed
```

Comment:
Need to update CI to give it a log file to log to and print its content on exit.